src/corehost/cli/deps_format.cpp

```diff
@@ -139,9 +139,9 @@ bool deps_json_t::perform_rid_fallback(rid_specific_assets_t* portable_assets, c
         assert(!matched_rid.empty());
         for (auto iter = package.second.begin(); iter != package.second.end(); /* */)
         {
-            iter = (iter->first != matched_rid)
-                 ? package.second.erase(iter)
-                 : iter++;
+            (iter->first != matched_rid)
```

Comment:
I'd prefer `if` here because we are not using return value of `?:` for anything.